sparse/client.go

```diff
@@ -327,15 +342,19 @@ func (client *syncClient) open() error {
 	}
 
 	// drain the buffer and close the body
-	_, _ = io.Copy(ioutil.Discard, resp.Body)
+	body, err := ioutil.ReadAll(resp.Body)
```

Comment:
Should use `io.ReadAll`, because iotuil has been deprecated.

Reply:
Updated. Thank you.